README.md

```diff
@@ -4,7 +4,7 @@
   <img src="https://user-images.githubusercontent.com/10407788/88133997-77560f00-cbb1-11ea-874c-a8f1d123a9df.jpg" style="max-width: 800px;" alt="rio-tiler"></a>
 </p>
 <p align="center">
-  <em>Rasterio plugin to read web map tiles from raster datasets.</em>
+  <em>User friendly Rasterio plugin to read raster datasets.</em>
```

Comment:
Seems a bit too much 🤷‍♂️ 

any ideas?

Reply:
I agree it's a bit much, but don't have other suggestions 🤷